lib/perl/Genome/Model/Tools/Analysis/DumpIgvXmlMulti.pm

```diff
@@ -97,7 +98,8 @@ XML
 
     my $i=0;    
     for($i=0;$i<@bams;$i++){
-        my $path = abs_path($bams[$i]);
+        my $uri = URI->new($bams[$i]);
+        my $path = $uri->scheme ? $bams[$i] : abs_path($bams[$i]);
```

Comment:
For some reason I'm not able to reproduce this behavior of abs_path anymore. I will do some more testing but you're right that this needs some more refined behavior.